### PR TITLE
Fix/repr bug

### PR DESCRIPTION
**Pull request by neyazbasheer**
#### Changes

Fixes # (issue)

#### Type of change

- [ ] 📚  Documentation Update
- [ ] 🧪 Tests Cases
- [x] 🐞 Bug fix (non-breaking change which fixes an issue)
- [ ] 🔬 New feature (non-breaking change which adds functionality)
- [ ] 🚨 Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] 📝 This change requires a documentation update

#### Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] Did you update CHANGELOG in case of a major change?

### CODE REVIEW

**Review by edgarriba — APPROVED** [review]
LGTM

File: kornia/feature/integrated.py [hunk 1]
```diff
@@ -82,17 +82,10 @@ def __init__(
...
-            + 'patch_size='
-            + str(self.patch_size)
-            + ', '
-            + 'grayscale_descriptor='
-            + str(self.grayscale_descriptor)
-            + ')'
+            f"{self.__class__.__name__}"
+            f"(descriptor={self.descriptor.__repr__()}, "
+            f"patch_size={str(self.patch_size)}, "
+            f" grayscale_descriptor='{str(self.grayscale_descriptor)})"
```
Comment (johnnv1) [inline kornia/feature/integrated.py:88]:
```suggestion
            f"patch_size={self.patch_size}, "
            f"grayscale_descriptor='{self.grayscale_descriptor})"
```

File: kornia/geometry/subpix/spatial_soft_argmax.py [hunk 2]
```diff
@@ -654,7 +618,7 @@ def __init__(self, strict_maxima_bonus: float = 10.0, eps: float = 1e-7) -> None
         return
 
     def __repr__(self) -> str:
-        return self.__class__.__name__ + '(' + 'strict_maxima_bonus=' + str(self.strict_maxima_bonus) + ')'
+        return f"{self.__class__.__name__}" f"strict_maxima_bonus={self.strict_maxima_bonus})"
```
Comment (johnnv1) [inline kornia/geometry/subpix/spatial_soft_argmax.py:621]:
```suggestion
        return f"{self.__class__.__name__}(strict_maxima_bonus={self.strict_maxima_bonus})"
```

**Review by johnnv1 — APPROVED** [review]
LGTM, just a small fix

**Pull request merged**